app/models/animations.py

```diff
@@ -97,9 +97,9 @@ def __init__(self, gif_path: str, target: Callable[..., Any]):
 
         logger.debug("Finished LoadingAnimation initialization")
 
-    def check_animation_stop(self, frameNumber: int) -> None:
+    def check_animation_stop(self, framenumber: int) -> None:
```

Comment:
If you are not going to camel case, please use `frame_number` with an underscore.

Reply:
yeah done thanks